<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>researchRemed</class>
 <widget class="QMainWindow" name="researchRemed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно исследования препаратов</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="repertoryFilter">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Полный реперторий</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="show">
        <property name="text">
         <string>Показать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="strategy">
        <property name="text">
         <string>Стратегия</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="opaqueResize">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="spacefiller" native="true"/>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget1">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QWidget" name="widgetFiller" native="true">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGraphicsView" name="graphicsView_header">
              <property name="frameShape">
               <enum>QFrame::Shape::NoFrame</enum>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
              </property>
              <property name="interactive">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
              </property>
              <property name="renderHints">
               <set>QPainter::RenderHint::Antialiasing|QPainter::RenderHint::TextAntialiasing</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QGraphicsView" name="graphicsView_count">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>0</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::NoFrame</enum>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
              </property>
              <property name="interactive">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGraphicsView" name="graphicsView_remedies">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::NoFrame</enum>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
              </property>
              <property name="interactive">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollBar" name="horizontalScrollBar">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="propertySymptom">
   <property name="text">
    <string>&quot;Редактировать - Свойства&quot;</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="deleteSymptom">
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="cutoutSymptom">
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="putSymptom">
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="copySymptomsText">
   <property name="text">
    <string>Копировать текст симптома</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="copySymptom">
   <property name="text">
    <string>Копировать симптом</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="enabledMeasure">
   <property name="text">
    <string>Выделенные степени</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>strategy</sender>
   <signal>clicked()</signal>
   <receiver>researchRemed</receiver>
   <slot>openStrategyMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>884</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show</sender>
   <signal>clicked()</signal>
   <receiver>researchRemed</receiver>
   <slot>openShowMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>798</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>splitter</sender>
   <signal>splitterMoved(int,int)</signal>
   <receiver>researchRemed</receiver>
   <slot>splitterMoved(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>researchRemed</receiver>
   <slot>openListMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalScrollBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>researchRemed</receiver>
   <slot>vScrollBarMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>918</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>researchRemed</receiver>
   <slot>hScrollBarMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>propertySymptom</sender>
   <signal>triggered()</signal>
   <receiver>researchRemed</receiver>
   <slot>openProperty()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteSymptom</sender>
   <signal>triggered()</signal>
   <receiver>researchRemed</receiver>
   <slot>deleteSymptom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cutoutSymptom</sender>
   <signal>triggered()</signal>
   <receiver>researchRemed</receiver>
   <slot>cutoutSymptom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>putSymptom</sender>
   <signal>triggered()</signal>
   <receiver>researchRemed</receiver>
   <slot>putSymptom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copySymptomsText</sender>
   <signal>triggered()</signal>
   <receiver>researchRemed</receiver>
   <slot>copySymptomsText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copySymptom</sender>
   <signal>triggered()</signal>
   <receiver>researchRemed</receiver>
   <slot>copySymptom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enabledMeasure</sender>
   <signal>triggered()</signal>
   <receiver>researchRemed</receiver>
   <slot>enabledMeasure()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>researchRemed</receiver>
   <slot>enableRepertoryFilter(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>repertoryFilter</sender>
   <signal>clicked()</signal>
   <receiver>researchRemed</receiver>
   <slot>showReperotyFilterMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openStrategyMenu()</slot>
  <slot>openShowMenu()</slot>
  <slot>splitterMoved(int,int)</slot>
  <slot>openListMenu(QPoint)</slot>
  <slot>hScrollBarMoved(int)</slot>
  <slot>vScrollBarMoved(int)</slot>
  <slot>openProperty()</slot>
  <slot>deleteSymptom()</slot>
  <slot>cutoutSymptom()</slot>
  <slot>putSymptom()</slot>
  <slot>copySymptomsText()</slot>
  <slot>copySymptom()</slot>
  <slot>enabledMeasure()</slot>
  <slot>enableRepertoryFilter(int)</slot>
  <slot>showReperotyFilterMenu()</slot>
 </slots>
</ui>
